src/config.h

```diff
@@ -330,7 +334,7 @@ void setcpuaffinity(const char *cpulist);
 #endif
 
 /* Test for posix_fadvise() */
-#if defined(__linux__) || __FreeBSD__ >= 10
+#if defined(__linux__) || defined(__FreeBSD__)
```

Comment:
Could you please explain why you changed it.

Reply:
freebsd 10 has been deprecated for over 10 years, freebsd 11 and 12 have also been deprecated over 6 years. i was told by the senior director of technology for the FreeBSD foundation that `__FreeBSD__ >= 10` should be changed to `defined(__FreeBSD__)` as a best practice when i was working on other freebsd projects.

Reply:
What you said makes sense. However, the minimum gcc version we support now is 4.9. I'm not sure if freebsd 10 meets it. We'll try our best to make the old toolchain compile successfully as well.
@oranagra WDYT?